src/travel-details-screens/components/FlexibleTransportBookingOptions.tsx

```diff
@@ -0,0 +1,93 @@
+import {Phone} from '@atb/assets/svg/mono-icons/devices';
+import {ExternalLink} from '@atb/assets/svg/mono-icons/navigation';
+import {useTranslation, TripDetailsTexts} from '@atb/translations';
+import {Leg} from '@atb/api/types/trips';
+import {View, Linking} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import {Button} from '@atb/components/button';
+import {BookingMethod} from '@atb/api/types/generated/journey_planner_v3_types';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+type FlexibleTransportBookingOptionsProps = {
+  leg: Leg;
+};
+
+export const FlexibleTransportBookingOptions: React.FC<
+  FlexibleTransportBookingOptionsProps
+> = ({leg}) => {
+  const {t} = useTranslation();
+  const style = useStyle();
+
+  const {theme} = useTheme();
+  const {bottom: safeAreaBottom} = useSafeAreaInsets();
+
+  const bookingArrangements = leg.bookingArrangements;
+
+  const bookingPhone = bookingArrangements?.bookingContact?.phone;
+  const bookingUrl = bookingArrangements?.bookingContact?.url;
+
+  const bookingMethods = bookingArrangements?.bookingMethods;
+
+  const showBookOnlineOption =
+    bookingUrl && bookingMethods?.some((bm) => bm === BookingMethod.Online);
+  const showBookByPhoneOption =
+    bookingPhone &&
+    bookingMethods?.some((bm) => bm === BookingMethod.CallOffice);
+
+  if (showBookOnlineOption || showBookByPhoneOption) {
```

Comment:
This condition might not be needed.

Reply:
Currently needed to avoid rendering this when no options show:
```
<View // 200% zoom bugfix for FlexibleTransportBookingDetails
   style={{height: Math.max(safeAreaBottom, theme.spacings.medium)}}
/>
```

Reply:
Update: following a change in the bugfix, it is indeed no longer needed :+1: